### ISSUE TITLE
Don't Allow more than 1 instance to be loaded at a time

### ISSUE THREAD

**Issue by sanford** [Davidobot/BetterJoy#328]
Issue:

Multiple BetterJoy's can be launched and they conflict with each other.

Suggested fix:

At startup if another instance of BetterJoy is loaded, exit the new instance.

**Comment by ceephis** [comment]
I have had this problem as well.  

When the program automatically disconnects a joycon or simply freezes when I go to check its settings or reconnect the joycon somehow I get multiple copies of the program running. 

Recently I was simply browsing the internet and my laptops fan was flying.  I checked my cpu usage and it was above 90%.  When I went to see what was using the CP I found 3 versions of Better joy running all using 25%+ of my i7.

is there a way to force only one version of the app to run at a time on my PC?  I often don't notice when or why another version will open.

Thank you for your communication and help
